tests/test_helper/common.bash

```diff
@@ -218,3 +218,42 @@ function netns_ifadd() {
     lxc_exec "$container" \
         "ip netns exec $name ip link set up dev $ifname"
 }
+
+# ping_start CONTAINER DST [ NETNS ]
+#
+# Start ping to DST in CONTAINER in the background, optionally using network
+# namespace NETNS.
+#
+# Output from ping and the PID will be recorded as files in '/tmp/' in
+# CONTAINER, and the process and results can be reaped by a subsequent call to
+# ``ping_reap``.
+function ping_start() {
+    local container=$1; shift
+    local dst=$1; shift
+    local netns=$1
+
+    local base_filename="${netns:+$netns-}${dst//[\.:]/_}"
```

Comment:
I wonder if it would be worth to turn the filename creation into a one-line function that both `ping_start` and `ping_reap` call. To keep it consistent.

Reply:
Yea, it's a lot of variable passing and echoing back though, which I think is why I opted not to.

But you're right consistency and correctness trumps that